components/plurals/src/operands.rs

```diff
@@ -34,33 +34,33 @@ use fixed_decimal::FixedDecimal;
 /// ```
 /// use icu::plurals::PluralOperands;
 /// assert_eq!(
-///     PluralOperands {
```

Comment:
Nit: Rewrite these tests to use only public APIs. It may mean that we don't assert the exact icwftc values any more. Wait for feedback on @zbraniecki on how to move forward.